charter.md

```diff
@@ -63,6 +63,30 @@ are able to proceed with a meeting.
 * Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
 * Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
 
+## Elections
+
+Every year, the Steering Committee holds an election for open seats with SIG 
+Contributor Experience owning the process. Our [election policy document](https://git.k8s.io/steering/elections.md) 
+covers the details for how this works.
+### Vacancies and Removal
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+#### Removal
+
+A Steering Committee member may be removed by a two-thirds vote of the other 
```

Comment:
> The steering committee has both private and public meetings. Removal should be required to be a public meeting, with public reasoning, and a public vote.

Considering that the removal process could become quite contentious and we can't foresee all of the possible situations that might lead to the removal process being invoked, especially in the case of things like CoC violations, I don't think we can expect that it will always be possible to conduct the removal process in a public forum.

That said, I do believe that there should be a public artifact produced that is made available providing relevant details to the community regarding the removal process.

Reply:
There are sometimes legal reasons to not disclose _why_ something has happened. CoCC actions are notably not public for this reason too.

Reply:
> Removal should be required to be a public meeting, with public reasoning, and a public vote.

Votes are always public, but sometimes might be done async on an issue. As for reasoning, each member might have slightly different reasoning as is the case any time we vote on an issue.

I'll note that this section doesn't mention and isn't related to the code of conduct. CoC violations are handled by the Code of Conduct committee, not Steering. This would be if, for any reason, the committee has no confidence in a member and feels that member is unfit to serve.

Reply:
+1 to @endocrimes' reasoning. The Committees are chartered with a certain level of privacy in mind. 

Any public requirement for removal for a member would end up either as a popularity contest in the project or a burning at the stake of the person in question.

Reply:
A secret tribunal to remove someone who was elected as a community representative for secret reasons, with a secret vote is not in keeping with community values around transparency.  If the action is important enough to remove the person from office (code of conduct or otherwise), it's important enough to explain why that must be done, how it is in the best interest of the community, and how community representatives cast their votes.

Reply:
I agree secrecy is not in line with our values. In fact, there may be legal antitrust implications - we should ask LF legal about that.

Reply:
@deads2k @johnbelamaric please scroll down to see the discussion below with Daniel and thockin.